common/constants.go

```diff
@@ -22,10 +22,10 @@ const (
 
 	DATABASE_CR_STATUS_DELETING     = "DELETING"
 	DATABASE_CR_STATUS_EMPTY        = ""
-	DATABASE_CR_STATUS_READY        = "READY"
+	DATABASE_CR_STATUS_NOT_FOUND    = "NOT FOUND"
 	DATABASE_CR_STATUS_PROVISIONING = "PROVISIONING"
-
-	DATABASE_DEFAULT_INSTANCE_NAME = "database_instance_name"
+	DATABASE_CR_STATUS_READY        = "READY"
+	DATABASE_CR_STATUS_WAITING      = "WAITING"
```

Comment:
Added for the transient state when a db provisioning has been requested on NDB and the NDBServer CR has not reconciled yet.